File: leetcode/412_fizz_buzz/sol.js
const fizzBuzz = n => {
  const res = [];

  let i = 1;
  while (i <= n) {
    if (i % 15 === 0) {
      res.push("FizzBuzz");
    } else if (i % 3 === 0) {
      res.push("Fizz");
    } else if (i % 5 === 0) {
      res.push("Buzz");
    } else {
      res.push(`${i}`);
    }
    i += 1;
  }

  return res;
};

File: leetcode/412_fizz_buzz/sol.rb
def fizz_buzz(n)
  res = []

  (1..n).each do |num|
    if num % 15 == 0
      res << "FizzBuzz"
    elsif num % 3 == 0
      res << "Fizz"
    elsif num % 5 == 0
      res << "Buzz"
    else
      res << "#{num}"
    end
  end

  res
end
